bika/lims/tests/base.py

```diff
@@ -31,15 +44,36 @@ def setUp(self):
         # During testing, CSRF protection causes failures.
         os.environ["PLONE_CSRF_DISABLED"] = "true"
 
-    def getBrowser(self, loggedIn=True):
+    def getBrowser(self,
+            username=TEST_USER_NAME,
+            password=TEST_USER_PASSWORD,
+            loggedIn=True):
         """ instantiate and return a testbrowser for convenience """
         browser = Browser(self.portal)
         browser.addHeader('Accept-Language', 'en-US')
         browser.handleErrors = False
         if loggedIn:
             browser.open(self.portal.absolute_url())
-            browser.getControl('Login Name').value = TEST_USER_NAME
-            browser.getControl('Password').value = TEST_USER_PASSWORD
+            browser.getControl('Login Name').value = username
+            browser.getControl('Password').value = password
             browser.getControl('Log in').click()
             self.assertTrue('You are now logged in' in browser.contents)
         return browser
+
+    def setup_data_load(self):
```

Comment:
@Espurna I'm pretty sure we should remove this.  Any tests which make use of the data in this file are brittle and prone to breaking when the data changes (which it always does), and so they should be creating their own data for each test locally.  It's also very slow to load, which is a barrier to running the tests locally.

@xispa what do you think of this?  I see that pau has ported a few tests that directly make use of this data.  It may be worth accepting this PR, but printing some extra comments in these tests (and maybe some logger warnings during run) to alert new testers to the deprecated nature of this xlsx file.

Reply:
I agree @rockfruit . I think adding some extra comments (or tag `setup_data_load` as deprectated) would be interesting. I suggest to keep the setupdata import for now (cause we want the tests to run again asap), and create the required objects properly (w/o setupdata) later, after we have integration merged into master. Once merged, we can remove `setup_data_load` function and force all tests to do it properly.